website/src/docs/hotchocolate/filtering/string-filter.md

```diff
@@ -0,0 +1,73 @@
+### String Filter
```

Comment:
Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.